_includes/header.html

```diff
@@ -41,8 +41,8 @@
                     <i class="fa fa-bars icon-custom"></i>
                 </button>
 
-                <a class="navbar-brand" href="index.html" title="Sterling Electrical Contractors of Seattle">
-                    <img src="images/sec-logo-white.png" alt="Sterling Electrical Contractors of Seattle">
+                <a class="navbar-brand" href="/" title="Sterling Electrical Contractors of Seattle">
+                    <img src="/images/sec-logo-white.png" alt="Sterling Electrical Contractors of Seattle">
```

Comment:
all paths that are generic like images should be variables. This makes it easier to reconfigure the location of everything on the fly.